common/document_parser/lib/section_parse/parsers/parser_factory.py

```diff
@@ -0,0 +1,36 @@
+from os.path import split
+from common.document_parser.lib.document import FieldNames
+from .parser_definition import ParserDefinition
+from .dod_parser import DoDParser
+from .navy_parser import NavyParser
+from .cjcs_parser import CJCSParser
+from .eo_parser import EOParser
+from .uscode_parser import USCodeParser
+
+class ParserFactory:
+    """Create a parser for a specific document type."""
+
+    @staticmethod
+    def create(doc_dict: dict) -> ParserDefinition:
+        """Create a parser for the document type defined in the document
+        dictionary.
+
+        Args:
+            doc_dict (dict): JSON representation of the document.
+
+        Returns:
+            ParserDefinition
+        """
+        doc_type = split(doc_dict[FieldNames.DOC_TYPE])[1].lower()
```

Comment:
@mishoe That's a problem since the type of parser is determined by the `doc_type` field.  
Since we are only focusing on CJCS for now, I'll leave this comment unresolved and we can come back to it when we revisit US Code docs. 

cc: @amaruca